docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -1 +1,326 @@
+
+NOTE - COULD TALK ABOUT THE SHAKA PSSH BOX CONVERSION ALSO WITHIN THE SHAKA PACKAGER DOCKER IMAGE??????.......
+
+PlayReady does not require a `pssh` value in order for it to be encrypted. This is handled by the key values set in Line 4.
+
+### Line 6 
+
+`protection_systems` is where you input which DRM you would like to encrypt. In this case, `Widevine, PlayReady` is entered.
+
+### Line 7
+
+`mpd_output` is where you name the manifest and where it is saved once packaged. In this case, `media/encrypted-test.mpd` is entered.
+
+## Packaging Content with Fairplay DRM
+
+Below is an example of how we can send a request to Shaka Packager to encrypt our content with Fairplay DRM and package it: 
+
+```
+  packager \
+  in=media/audio.mp4,stream=audio,output=media/encrypted-test-hls-audio.mp4,drm_label=AUDIO \
+  in=media/test.mp4,stream=video,output=media/encrypted-test-hls-video.mp4,drm_label=VIDEO \
+  --protection_scheme cbcs \
+  --enable_raw_key_encryption \
+  --keys label=AUDIO:key_id=<key_id_hex_value>:key=<content_key_hex_value>,label=VIDEO:key_id=<key_id_hex_value>:key=<content_key_hex_value> \
+  --protection_systems Fairplay \
+  --iv <iv_hex_value> \
+  --hls_master_playlist_output media/encrypted-test-hls.m3u8 \
+  --hls_key_uri skd://fairplay-license.vudrm.tech/v2/license/<content_id>
+```
+
+Packaging content with Fairplay DRM is much of the same as packaging with Widevine and PlayReady DRM. However, there are differences detailed below.
+
+### Line 3
+
+`protection_scheme` must be set to `cbcs`
+
+### Line 6
+
+`protection_systems` is now set to Fairplay.
+
+### Line 7
+
+The `iv_hex_value` from the "CPIX keys as JSON" document is needed.
+
+### Line 8
+
+`hls_master_playlist_output` acts similar to the `mpd_output`. The file name must have the suffix `.m3u8`.
+
+### Line 9
+
+`hls_key_uri` is set to the value of the `fairplay_laurl` in the "CPIX keys as JSON" document.
+
+## Testing packaged content
+
+Upload your packaged files and manifest to AWS S3 or an equivalent hosting service.
+
+You can now test your newly packaged content with our player on the VUDRM Admin site.
+
+- Log on to the VUDRM Admin site - https://admin.vudrm.tech
+
+- On the left hand side of the page select `Configuration`
+
+- Click on `Test your Stream` 
+
+- A token is automatically generated for use with your content
+ 
+- If you are testing Widevine / PlayReady content, enter the URL for where your manifest has been uploaded in the `DASH Stream URL` text box
+
+- If you are testing Fairplay content, ensure you are using a Mac and using Safari as your browser, enter the URL for where your manifest has been uploaded in the `HLS Stream URL` text box
+
+- Click `Load Player`
+
+- The player should now appear, click play to play the content
+
+
+NOTE - TALK ABOUT HOW TO SEE IF LICENSES HAVE BEEN AQUIRED IN THE NETWORK TABS FOR EACH BROWSER???....
```

Comment:
This can be said but should be brief, something to the tune of "You can confirm a relevant license request has been made by checking the network tab of the browsers dev tools"

Reply:
Sounds good to me.

Reply:
Actioned.